ui/src/components/break-line/QasBreakLine.spec.js

```diff
@@ -0,0 +1,32 @@
+import { mount } from '@vue/test-utils'
+
+import '@/index.js'
+import QasBreakLine from './QasBreakLine.vue'
+
+describe('Test QasBreakLine component', () => {
+  const wrapper = mount(QasBreakLine, {
+    propsData: {
+      text: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. \n Error tempora vero aspernatur facere. Deserunt delectus amet eveniet velit reprehenderit itaque',
+      tagClass: 'my-class'
+    }
+  })
+
+  it('test defaults', () => {
+    expect(wrapper.element).toMatchSnapshot()
+    expect(wrapper.findAll('div')).toHaveLength(3)
+    expect(wrapper.findAll('.my-class')).toHaveLength(2)
+    expect(wrapper.isVisible()).toBe(true)
+  })
+
+  it('change parentTag', async () => {
+    await wrapper.setProps({ parentTag: 'section' })
+
+    expect(wrapper.find('section').isVisible()).toBe(true)
+  })
+
+  it('change tag', async () => {
+    await wrapper.setProps({ tag: 'span' })
+
+    expect(wrapper.findAll('span')).toHaveLength(2)
+  })
```

Comment:
Adicionar teste para a quebra de linha e para a adição de style

Reply:
Mas para a quebra de linha é justamente isto que estou fazendo com o span, se te 2 é porque quebrou eles

Reply:
A ideia não é passar uma string ou um code para fazer a quebra de linha? no caso que esta no storybook, você pede para quebrar a linha no `/n`... eu digo... se mudar isso para um `.` como o code ficaria?

ou seja.... você possui lá no code uma condição que passa dentro dessa prop... faz o teste que sessa prop for diferente, como ficaria